src/quo2/components/communities/community_view.cljs

```diff
@@ -0,0 +1,86 @@
+(ns quo2.components.communities.community-view
+  (:require
```

Comment:
split the community card out into 3 parts, the core code and it's 2 use cases. 
The core code I will reuse in the following pr for the community landing page.